mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.stories.js

```diff
@@ -34,8 +34,25 @@ export const OtherOptionPrePopulated = () => (
   <CheckboxGroup
     id='bar'
     options={options}
-    value={{ selectedValues: ['Option 1', 'Option 2'], otherValue: 'Some other value' }}
+    value={{
+      selectedValues: [{ choice: 'Option 1' }, { choice: 'Option 2' }],
+      otherValue: { choice: 'Some other value' }
+    }}
     onChange={action('onChange')}
     shouldAddOtherOptionWithClarification={true}
   />
 )
+
+export const WithNestedMarkupForSelected = () => (
+  <CheckboxGroup
+    options={options}
+    value={{
+      selectedValues: [{ choice: 'Option 1' }, { choice: 'Option 2' }],
+      otherValue: { choice: 'Some other value' }
+    }}
+    onChange={action('onChange')}
+    id='cuddlyKittens'
+    shouldAddOtherOptionWithClarification={true}
+    selectedMarkup={<>Youve selected ME</>}
```

Comment:
selectedMarkup allows the parent to inject and controll nested questions. the component will append an unique `optionId` for each iteration of the selectedMarkup (it gets applied to each option if selected)